build.gradle.kts

```diff
@@ -113,11 +155,21 @@ fun CopySpec.fromCompileDependencies() {
 }
 
 val pluginZip by tasks.creating(Zip::class) {
-    dependsOn(assemblePlugin)
+    dependsOn(tasks.assemble)
+    dependsOn(tasks.getByName("generateLicenseReport"))
 
-    fromCompileDependencies()
-    into(pluginId)
-    archiveBaseName.set(pluginName)
+//    fromCompileDependencies()
+//    into(pluginId)
+    from(tasks.assemble.get().outputs.files)
+    from(extensionJsonFile)
+    from("src/main/resources") {
+        include("dependencies.json")
+    }
+    from("src/main/resources") {
+        include("icon.svg")
+        rename("icon.svg", "pluginIcon.svg")
```

Comment:
I see a mix of use of the word extension and plugin. It could be the JetBrains thing. I am just pointing it out.

Reply:
Yes, definitely JetBrains.